firmware/baseband/proc_btlerx.cpp

```diff
@@ -95,7 +95,7 @@ void BTLERxProcessor::scramble_byte(uint8_t* byte_in, int num_byte, const uint8_
     }
 }
 
-int BTLERxProcessor::parse_adv_pdu_payload_byte(uint8_t* payload_byte, int num_payload_byte, ADV_PDU_TYPE pdu_type, void* adv_pdu_payload) {
```

Comment:
Where did all that decoding code went ?
It isn't needed anymore ?

Reply:
It's because the code was doing nothing as we directly cast data into packets structures.